### PR TITLE
Feature/nav 173 results and discussion limitations

### PR DESCRIPTION
**Pull request by munterfi**
A first version of the limitations.

@Brunner246 Can you check if the last two parapgraphs about the RAPTOR in C++ are correct?

### CODE REVIEW

**Comment by munterfi** [comment]
PS: We are not able to route on the GTFS of germany... somehow there are many invalid references. But it could also be a bug on our side. I would suggest fixing this or omit this in the thesis.

image

**Comment by Brunner246** [comment]
@munterfi i would not start refactoring/bugfixing 2 days before submition 🙃

> PS: We are not able to route on the GTFS of germany... somehow there are many invalid references. But it could also be a bug on our side. I would suggest fixing this or omit this in the thesis.
> 
> image

File: Writerside/topics/implementation/raptor-xx.md [hunk 1]
```diff
@@ -359,8 +359,8 @@ The Visual Studio Diagnostic Tools were utilized to analyze and optimize the per
 ## Foreign Function and Memory (FFM) API
 
 Initially, we explored the possibility of calling the C++ RAPTOR implementation from Java using the Foreign Function and
-Memory (FFM) API. However, we faced significant challenges due to the complexity of mapping intricate data structures
-between C++ and Java. Ultimately, we decided against this approach.
+Memory (FFM) API [7]. However, we faced significant challenges due to the complexity of mapping intricate data
+structures between C++ and Java. Ultimately, we decided against this approach.
 
 The FFM API (Foreign Function & Memory) was not well suited for our application. To send routing requests from Java to
 C++, C++ had to interact with a memory structure created in Java, which would have been very complex to implement. Since
```
[the remark this answers is not in the document]
Comment (munterfi) [reply Writerside/topics/implementation/raptor-xx.md:366]: Too late: https://github.com/naviqore/public-transit-service/pull/122

[the remark this answers is not in the document]
Comment (munterfi) [reply Writerside/topics/implementation/raptor-xx.md:366]: This sentence is correct, but I agree... it is not very flowing while reading

**Review by clukas1 — APPROVED** [review]
Good from my side.

**Review by Brunner246 — APPROVED** [review]

**Pull request merged**